src/scancode/cli.py

```diff
@@ -949,8 +949,11 @@ def echo_func(*_args, **_kwargs):
         codebase.counters['final:files_count'] = files_count
         codebase.counters['final:dirs_count'] = dirs_count
         codebase.counters['final:size_count'] = size_count
+        
+        processing_end = time()
```

Comment:
Why do you need an intermediate variable?
May be you could directly use `time()` in https://github.com/nexB/scancode-toolkit/pull/1942/files#diff-1e37ecfb9a8c3fccd8ecdfb2ce51f591R956 ?